packages/apps/tools/src/context/connect-wallet-context.tsx

```diff
@@ -72,14 +109,15 @@ export const WalletConnectClientContextProvider: FC<
   );
 
   const connect = useCallback(
-    async (pairing?: { topic: string }) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    async (pairing: any) => {
```

Comment:
There is no way of typing this parameter?

Reply:
not really, tbh i tried everything i knew and everytime it was giving me a different error